### ISSUE TITLE
standalone with WP - Use of undefined constant OWA_ERROR_HANDLER

### ISSUE THREAD

**Issue by dac2020** [padams/Open-Web-Analytics#226]
[Sun Jan 22 17:02:27.913970 2017] [:error] [pid 21253] [client x.x.x.x:64422] PHP Notice:  Use of undefined constant OWA_ERROR_HANDLER - assumed 'OWA_ERROR_HANDLER' in /var/www/vhosts/owa/html/modules/base/templates/config_dom.tpl on line 15, referer: http://owa.domain.com/index.php?owa_do=base.optionsGeneral

I realize that this is just a notice but I am not seeing stats being generated for a wordpress site.

I installed a standalone master, got it working.
I then installed a copy of that as a plugin on a wordpress site. 
I enabled the plugin after making sure the owa-config.php file contained the same db info since the docs say to simply use the same db as the standalone and this should work.

It's been several hours, I can see traffic by watching the logs but never see any stats.
Is the above error related? There are no other errors either in owa logs or the server.

**Comment by dac2020** [comment]
Other than the above error, no other errors anywhere and no logs/starts what so ever after 15 hours.

**Comment by padams** [comment]
Turn on WordPress debugging and see if there are any errors coming from the plugin.

**Comment by dac2020** [comment]
There are a few errors but none that seem to be related to owa.

`Notice: wp_enqueue_script was called incorrectly. Scripts and styles should not be registered or enqueued until the wp_enqueue_scripts, admin_enqueue_scripts, or login_enqueue_scripts hooks. Please see Debugging in WordPress for more information. (This message was added in version 3.3.0.) in 
/var/www/html/wp-includes/functions.php on line 4136

Notice: has_cap was called with an argument that is deprecated since version 2.0.0! Usage of user levels by plugins and themes is deprecated. Use roles and capabilities instead. in /var/www/html/wp-includes/functions.php on line 4020

Notice: Undefined index: configuration in /var/www/html/wp-content/plugins/owa/modules/base/optionsGeneral.php on line 82

Notice: Undefined index: c in /var/www/html/wp-content/themes/Divi-Child/footer.php on line 52
Notice: ob_end_flush(): failed to send buffer of zlib output compression (1) in /var/www/html/wp-includes/functions.php on line 3718
Notice: get_settings is deprecated since version 2.1.0! Use get_option() instead. in /var/www/html/wp-includes/functions.php on line 3828
`

**Comment by padams** [comment]
what version of PHP and OS are you running under?

maybe check your permissions on the owa directory. The install needs to write out the config file there before creating tables. 
activating the plugin in wp again is an idem potent operation so it will simply try to run the install again without breaking anything.

**Comment by dac2020** [comment]
Os is Centos7 and php70w.

I set permissions when I install a new site so did that when installing owa standalone.
I used the same copy after it was installed stand alone so that I would know it's a working copy. 
No issues installing standalone. 

I've deactivated and reactivated a few times now. No difference. 
The site ID is correct, everything seems right but no stats.

I wish you were reading something from me that made you go 'doh, that's the problem' :).

**Comment by dac2020** [comment]
Os is Centos7 and php70w.

I set permissions when I install a new site so did that when installing owa standalone.
I used the same copy after it was installed stand alone so that I would know it's a working copy. 
No issues installing standalone. 

I've deactivated and reactivated a few times now. No difference. 
The site ID is correct, everything seems right but no stats.

I wish you were reading something from me that made you go 'doh, that's the problem' :).
